draft-ietf-rats-endorsements.md

```diff
@@ -201,13 +201,33 @@ some minimal matching policy (e.g., exact match against a singleton reference
 value).  This unfortunately complicates design as a Verifier may need
 multiple parsers for matching policies.
 
-# Endorsing Identity
+# Key Material for Attestation
 
-One type of claims that might be endorsed would be claims having to do with
-identity, such as verification keys bound to an identifier for the Attesting Environment (e.g., {{-rats-eat}} `ueid`).  While identity claims are just another
+A Verifier receives Evidence from an Attester that must be considered untrusted
+until established otherwise. This trust is established using cryptography. Typically,
+the bottom-most Attesting Environment in an Attester will sign claims about a Target Environment
+with a private key that that Attesting Environment possesses and the Verifier will verify
+them with a public key it possesses, called a verification key below. While this is typical,
+cryptography other than public key may also be used.
+
+Since it is not possible to establish trust in an Attester without cryptography,
+the Verifier must have a verification key for each Attester. Typically, the verification
+key comes to the Verifier in an Endorsement from an Endorser.
+
+No particular algorithm or cryptographic protocol is assumed for the verification
+of the Attester. The verification key could be a symmetric key, a public key, or
+something more sophisticated.
```

Comment:
What do we mean by _"more sophisticated"_?

Reply:
Some new DAA crypto protocol.

Reply:
awesome, should we add it as an example?

Reply:
I was thinking an example of "more sophistocated" would be a certificate.